content/news/050/index.md

```diff
@@ -76,56 +123,39 @@ If needed, a section can be split into subsections with a "------" delimiter.
 [Way of Rhea][wor] just got a release date: it will be coming to Steam on
 **May 20th, 2024**!
 
-Way of Rhea is a color based puzzle game with difficult puzzles, but forgiving
-mechanics being developed by [@masonremaley] in a custom Rust engine.
+Way of Rhea ([Steam](https://store.steampowered.com/app/1110620/Way_of_Rhea/)) is a color based puzzle game with difficult puzzles, but forgiving
```

Comment:
I think it's fine to have "inline links", but as historically **ALL** links were defined with a label I'm just noting this to start the discussion, just so we're clear about the expectations.

Reply:
Yeah, I noticed that too. Agreed, it's good to point this out.
I removed the lints for that intentionally because it doesn't matter for markdown that is not maintained after the post goes up. I can change it back if other maintainers disagree, but I'd prefer to be relaxed about this.